### PR TITLE
New Package: Rustlang.Rustup version 1.24.3

### PR DESCRIPTION
**Pull request by Craftzman7**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/64733)

### CODE REVIEW

**Comment by Craftzman7** [comment]
Don't know why it's complaining about that. The package URL and the installer URL belong to the same root domain.

File: manifests/r/Rustlang/Rustup/1.24.3/Rustlang.Rustup.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+PackageIdentifier: Rustlang.Rustup
+PackageVersion: 1.24.3
+PackageLocale: en-US
+Publisher: Rustlang
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+Author: Rustlang
+PackageName: Rustup
```
Comment (ItzLevvie) [inline manifests/r/Rustlang/Rustup/1.24.3/Rustlang.Rustup.locale.en-US.yaml:12]:
```suggestion
PackageName: "Rustup: the Rust toolchain installer"
```
This must match Control Panel.

File: manifests/r/Rustlang/Rustup/1.24.3/Rustlang.Rustup.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+InstallModes:
+- silent
+- interactive
+InstallerSwitches:
+  Silent: -y -q
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://static.rust-lang.org/rustup/dist/x86_64-pc-windows-msvc/rustup-init.exe
+  InstallerSha256: 780B445EE18BDC5A31AD6A4A63943927E5C0BBA8BFF664E8ECC8D5BCA3906C7B
```
Comment (ItzLevvie) [inline manifests/r/Rustlang/Rustup/1.24.3/Rustlang.Rustup.installer.yaml:17]:
```suggestion
  InstallerSha256: 780B445EE18BDC5A31AD6A4A63943927E5C0BBA8BFF664E8ECC8D5BCA3906C7B
  ProductCode: Rustup
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by HenkPoley** [comment]
I notice this package now keeps listing rustup as updateable, after updating rustup to 1.24.3

```
> winget upgrade
Name                                 Id              Version Available Source
-----------------------------------------------------------------------------
Rustup: the Rust toolchain installer Rustlang.Rustup Unknown 1.24.3    winget
1 upgrades available.
> rustup --version
rustup 1.24.3 (ce5817a94 2021-05-31)
info: This is the version for the rustup toolchain manager, not the rustc compiler.
info: The currently active `rustc` version is `rustc 1.62.0 (a8314ef7d 2022-06-27)`
```

**Pull request merged**